### PR TITLE
Establish build environment

### PR DESCRIPTION
**Pull request by wjcloud**
What does it do?
+ Change names for compiling dist file (now if you execute "make dist", it will create a tarball with "bitcoin-gold-XXX"
+ Clean up some pgp keys used from Core.
+ Delete lxcbr0 because it errors.

### CODE REVIEW

**Comment by wjcloud** [comment]
LOL same build error with cash hehe https://travis-ci.org/cculianu/bitcoin-abc/builds/268733782?utm_source=github_status&utm_medium=notification

**Comment by wjcloud** [comment]
Now it is passed... I will execute some builds.

**Comment by wjcloud** [comment]
It is mergable with this PR https://github.com/BTCGPU/BTCGPU/pull/221/commits/98a1a51cecf962534723a3f3a28d624048e9b896

**Comment by wjcloud** [comment]
Done.

**Comment by wjcloud** [comment]
One more thing: we need a release tag with v in front of it in order to build this easily.

**Comment by wjcloud** [comment]
Now it is clean! Rebased the master branch!

**Comment by wjcloud** [comment]
Added some commits originally from Bitcoin Core

**Comment by h4x3rotab** [comment]
Why did you remove `contrib/gitian-descriptors/README.md`?

File: contrib/gitian-descriptors/gitian-win.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ script: |
 
   function create_per-host_linker_wrapper {
   # This is only needed for trusty, as the mingw linker leaks a few bytes of
-  # heap, causing non-determinism. See discussion in https://github.com/bitcoin/bitcoin/pull/6900
+  # heap, causing non-determinism. See discussion in https://github.com/BTCGPU/BTCGPU/pull/6900
```
Comment (h4x3rotab) [inline contrib/gitian-descriptors/gitian-win.yml]: The URL shouldn't be changed: `https://github.com/bitcoin/bitcoin/pull/6900`

Comment (wjcloud) [reply contrib/gitian-descriptors/gitian-win.yml]: LOL, I've missed it.

**Comment by wjcloud** [comment]
@h4x3rotab  Because contrib/gitian-descriptors/README.md is a duplicate guide with doc/release-process.md and it is also outdated.

**Comment by wjcloud** [comment]
LOL now that's wierd, I had just fixed "comment" line...

**Comment by h4x3rotab** [comment]
lgtm

**Pull request merged**